tendrl/commons/objects/__init__.py

```diff
@@ -162,6 +162,16 @@ def save(self, update=True, ttl=None):
                 NS._int.wreconnect()
                 NS._int.wclient.write(item['key'], item['value'], quorum=True)
 
+    def load_all(self):
+        value = '/'.join(self.value.split('/')[:-1])
+        etcd_resp = NS._int.client.read(value)
```

Comment:
@anmolbabu True, I will create a wrapper in tendrl/commons/utils which will have read/write function  that can be called and will handle generalised exceptions